packages/modules/devices/sungrow/version.py

```diff
@@ -3,5 +3,6 @@
 
 class Version(IntEnum):
     SH = 0
-    SG = 1
-    SG_winet_dongle = 2
+    SH_winet_dongle = 1
+    SG = 2
+    SG_winet_dongle = 3
```

Comment:
```suggestion
    SG = 1
    SG_winet_dongle = 2
    SH_winet_dongle = 3
```
Die vorhandenen Versionsnummern am besten nicht mehr ändern, das verkompliziert nur.